components/image-upload.tsx

```
@@ -165,12 +183,18 @@ export function ImageUpload({
           {value.map((url, index) => {
             const isVideo =
               url.split(".").pop()?.toLowerCase() &&
-              ["mp4", "webm", "mov"].includes(url.split(".").pop()!.toLowerCase())
+              ["mp4", "webm", "mov"].includes(
+                url.split(".").pop()!.toLowerCase(),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)